cli/src/commands/invite/shared_recovery.rs

```diff
@@ -0,0 +1,91 @@
+// Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
+
+use libparsec::{InvitationEmailSentStatus, InvitationType, ParsecInvitationAddr};
+
+use crate::utils::*;
+
+crate::clap_parser_with_shared_opts_builder!(
+    #[with = config_dir, device, password_stdin]
+    pub struct Args {
+        /// Claimer email (i.e.: The invitee)
+        #[arg(long)]
+        email: String,
```

Comment:
What do you think of providing the email as a argument over an option ?

Reply:
Does our `clap_parser_with_shared_opts_builder` allow to do that?